controllers/featureflagconfiguration_controller.go

```diff
@@ -116,8 +117,23 @@ func (r *FeatureFlagConfigurationReconciler) Reconcile(ctx context.Context, req
 		}
 	}
 
-	// Update ConfigMaps
 	for _, cm := range ffConfigMapList {
+		// Append OwnerReference if not set
+		if !r.CheckOwnerReference(ffconf, cm) {
```

Comment:
```suggestion
if !r.featureFlagResourceIsOwner(ffconf,cm) {
```

can we have a method name more in line with what's going on? It's not a generic owner reference check, it's very specific to the feature flag CR>

Reply:
fixed